Assets/FallingPlatform.cs

```diff
@@ -0,0 +1,75 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FallingPlatform : MonoBehaviour
+{
+    // Start is called before the first frame update
+    [SerializeField]
+    GameObject spriteObject;
+    [SerializeField]
+
+    float elapsed = 0.0f;
+    [SerializeField]
+
+    float duration = 1.5f;
+    [SerializeField]
+
+    float magnitudeX = 0.3f;
+    [SerializeField]
+
+    float magnitudeY = 0.3f;
+    [SerializeField]
```

Comment:
could we call these `shakeMagnitudeX` and `shakeMagnitudeY` to make it painfully obvious what these are for? I read this and my first question is, "magnitude of what?"

Reply:
Fixed